examples/ExamplePdf.md

```diff
@@ -0,0 +1,191 @@
+# Mega Überschrift 
```

Comment:
Probably unwise to include the .md even as the biz logic of the examples changes over time

Reply:
Noted. In this case, what are some pdf example this repo is meant for then?

Reply:
It's meant to cover as many as possible, this fork was originally tailored for the parsing of judgments found on Singapore Law Watch